easybib/providers/nginx.rb

```diff
@@ -41,6 +41,11 @@
 
   ::Chef::Resource.send(:include, PhpFpm::Helper)
 
+  execute 'nginx_configtest' do
```

Comment:
This will likely collide in a loop.

```
excute_name = "nginx_config_#{config_name}"
execute execute_name do
  command '...'
  ...
end
```

Then notify on `"execute[#{execute_name}]"` below.